src/server/src/controllers/project.controller.ts

```diff
@@ -0,0 +1,41 @@
+import { Router } from 'express';
+import { ProjectService } from 'services/project.services';
+import { Request, Response } from 'express';
+import { createErrorResponse } from 'utils';
+
+const projectController = (router: Router): void => {
+  const projectService = new ProjectService();
+
+  router.get('/projects', async function (req: Request, res: Response) {
+    const projects = await projectService.getProjects();
+    console.log('RouterGet Projects:', projects);
+
+    // For each project, get the associated personnel, and then assign them to a casted
+
+    res.json(projects);
+  });
+
+  router.get('/projects/:id', async function (req: Request, res: Response) {
+    try {
+      const project = await projectService.getProjectById(req.params.id);
+      return res.send(project);
+    } catch (e) {
+      res.status(500);
+      return res.send(createErrorResponse(e));
+    }
+  });
+
+  router.post('/projects', async function (req: Request, res: Response) {
+    console.log('Router post project request');
```

Comment:
consider adding console logs to each of your controller endpoints.  You are very inconsistent with them now.  There should probably be a console.log/console.debug call for each, and if there's an error, there should be a console.error